index.html

```diff
@@ -9,36 +9,238 @@
-        <h2>I'am a Software Developer</h2>
-        <p>
-          I can help you build a product. featuer or <br />
-          website Look through some of my work <br />
-          and experience! if you see <br />
-          and have a project you need coded. <br />
-          don't hestiate to contact me.
-        </p>
-        <ul>
-          <li><a href=""><img src="images/Vector3.png" alt="" id="footer-img" /></a></li>
-          <li><a href=""><img src="images/linkedin.png" alt="" id="footer-img"/></a></li>
-          <li><a href=""><img src="images/Vector2.png" alt="" id="footer-img" /></a></li>
-          <li><a href=""><img src="images/Vector1.png" alt="" id="footer-img" /></a></li>
-          <li><a href=""> <img src="images/Vector.png" alt="" id="footer-img" /></a></li>
-        </ul>
-        <img src="images/illustration-header-mobile.png" class="side-img" alt=""/>
-      </section>
+    <div class="work-section">
+      <div class="wrapper">
+        <header>
+          <div class="right-side">
+            <a href="#"><div class="logo">Anicet Fantômas</div></a>
+          </div>
+          <nav>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+          </nav>
+        </header>
+        <section>
+          <h1>Hey There.<br />I'm Anicet</h1>
+          <h2>I'am a Software Developer</h2>
+          <p>
+            I can help you build a product. featuer or <br />
+            website Look through some of my work <br />
+            and experience! if you see <br />
+            and have a project you need coded. <br />
+            don't hestiate to contact me.
+          </p>
+          <ul>
+            <li>
+              <a href=""
+                ><img src="images/Vector3.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img src="images/linkedin.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img src="images/Vector2.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href=""
+                ><img src="images/Vector1.png" alt="" id="footer-img"
+              /></a>
+            </li>
+            <li>
+              <a href="">
+                <img src="images/Vector.png" alt="" id="footer-img"
+              /></a>
+            </li>
+          </ul>
+          <img
+            src="images/illustration-header-mobile.png"
+            class="side-img"
+            alt=""
+          />
+        </section>
+      </div>
+      <div class="container00 container1">
+        <h1 class="content-title">My Recent Works</h1>
+        <div class="img3"><img src="./images/img3.png" alt="" /></div>
+        <div class="content1">
+          <div class="content-title2">Multi-Post Stories</div>
```

Comment:
As the previous reviewer requested, please, kindly use a `hx` element instead of a `div` element.

Please, see my comment above about HTML5 semantic tags.

```suggestion
          <h2 class="content-title2">Multi-Post Stories</h2>
```

Please, kindly do so for all work cards.